starter-code/iteration2/iteration2/src/app/components/main/main.component.html

```diff
@@ -0,0 +1,5 @@
+<div class="container">
+  <h1>{{title}}</h1>
+  <img src="https://thumbs.dreamstime.com/b/main-street-via-laietana-barcelona-name-major-thoroughfare-lots-shops-attractions-around-61564902.jpg">
+  <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer blandit et dolor eu ultricies. Donec a pretium urna. Nunc ut mi id est vulputate dapibus nec sit amet dolor. In dui neque, bibendum non mi quis, tristique finibus eros. Curabitur cursus vulputate sollicitudin. Nulla eget consequat massa. Fusce imperdiet lectus id magna pellentesque, quis tristique sapien posuere. Vestibulum leo risus, dictum vel nisi et, sodales dictum ex. Quisque aliquam lectus lobortis, rhoncus nibh non, rhoncus tortor. Fusce vitae blandit nulla. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae;</p>
```

Comment:
It's ok but you can pass all the data (img src and description) as you pass the title.